### PR TITLE
More flexible reporting

### PR DESCRIPTION
**Pull request by loicdtx**
Reporting of the monitoring status was very static in the previous version. Allowing only to return the mask and disturbance_date without any option for the user to control layers returned by the `.report()` method. `.report()` now has a `layers=` argument which enables the user to select the desired layers.

This change breaks backward compatibility of the private `._report()` method

This change alone would immediately be released in a new version (0.5.0)

### CODE REVIEW

**Comment by loicdtx** [comment]
@jonasViehweger can you have a quick look at that?

File: nrt/monitor/__init__.py [hunk 1]
```diff
@@ -322,7 +353,6 @@ def report(self, filename, driver='GTiff', crs=CRS.from_epsg(3035)):
                 'height': r.shape[-2],
                 'width': r.shape[-1]}
         with rasterio.open(filename, 'w', **meta) as dst:
-            # TODO: Not sure this will work for single band without passing 1 to read()
             dst.write(r)
```
Comment (jonasViehweger) [inline nrt/monitor/__init__.py:356]:
```suggestion
            dst.descriptions = layers
            dst.write(r)
```

**Comment by jonasViehweger** [comment]
Looks good to me, I only suggested a line to add the names of the layers as band names to the tif.
I tested it and it should work as is.

File: nrt/monitor/__init__.py (hunk 1 above)
Comment (loicdtx) [reply nrt/monitor/__init__.py:356]: Thanks, very nice suggestion

**Pull request merged**